Sources/App/RepoMapping.swift

```diff
@@ -3,24 +3,40 @@ import Vapor
 
 struct RepoMapping {
     let repository: GitHubService.Repository
-    let environment: JiraService.CRPIssueFields.Environment
-    let jiraSummary: (GitHubService.Release) -> String
+    let crp: CRP
+
+    struct CRP {
+        let environment: JiraService.CRPIssueFields.Environment
+        let jiraSummary: (GitHubService.Release) -> String
+    }
 }
 
 extension RepoMapping {
     // [CNSMR-1319] TODO: Use a config file to parametrise repo list
-    static let all: [String: RepoMapping] = [
-        "ios": RepoMapping(
-            repository: GitHubService.Repository(fullName: Environment.get("GITHUB_REPO_IOS")!, baseBranch: "develop"),
+    static let ios = RepoMapping(
+        repository: GitHubService.Repository(
+            fullName: "Babylonpartners/babylon-ios",
```

Comment:
I thought we should parametrize as much as possible (using Env vars for now, config file later) and avoid having our private repos in here since Stevenson is a public OSS repo meant to be used by others in general?

Reply:
It's not a sensitive private info - repos are private anyway and without access token there is no way to access them. Internal services URLs, access tokens and other credentials.

Reply:
And even if they will be config file - this file will be in the repo (unless we go with storing it in a database for which we need to build bot interface)